website/views/options/social/chat-message.jade

```diff
@@ -31,6 +31,9 @@ mixin chatMessages(inbox)
         | &nbsp; &nbsp;
         a(ng-click="flagChatMessage(group._id, message)", ng-if=':: user.contributor.admin || (!message.sent && user.flags.communityGuidelinesAccepted && message.uuid != user.id && message.uuid != "system")')
           span.glyphicon.glyphicon-flag(tooltip="{{message.flags[user._id] ? env.t('abuseAlreadyReported') : env.t('abuseFlag')}}" ng-class='message.flags[user._id] ? "text-danger" : ""')
+        | &nbsp; &nbsp;
+        a(ng-click="copyToDo(message)")
```

Comment:
If you want to leave the option on for system messages, we should at least remove the part about the user. Right now it names the user "undefined" and gives a broken link to the user modal with the id `system`